bundles/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/internal/HarmonyHubGenericBindingProvider.java

```diff
@@ -105,12 +104,7 @@ public void processBindingConfiguration(String context, Item item, String bindin
 			HarmonyHubBindingConfig config = new HarmonyHubBindingConfig(type,qualifier,type.getLabel(),param1,param2, item.getClass());
 
 			addBindingConfig(item, config);		
-			Set<Item> items = contextMap.get(context);
-			if (items == null) {
-				items = new HashSet<Item>();
-				contextMap.put(context, items);
-			}
-			items.add(item);
+			super.processBindingConfiguration(context, item, bindingConfig);
 		} else {
```

Comment:
Removed code duplicated what super.processBindingConfiguration had already done, so moved super call to end.